neat-screen.js

```diff
@@ -338,10 +338,24 @@ NeatScreen.prototype.initializeCabalClient = function () {
   this.state.cabal = details
   this.state.messageScrollback = 0
   this.state.userScrollback = 0
-  var counter = 0
   this.client.getCabalKeys().forEach((key) => {
-    welcomeMessage.map((m) => this.client.getDetails(key).addStatusMessage({ timestamp: Date.now() + counter++, text: m }), '!status')
+    welcomeMessage.map((m) => this.client.getDetails(key).addStatusMessage({ text: m }), '!status')
+    this.state.moderationKeys = moderationKeys = this.state.cabal.core.adminKeys.map((k) => { return { key: k, type: 'admin' }}).concat(this.state.cabal.core.modKeys.map((k) => { return { key: k, type: 'mod' }}))
+    if (this.state.moderationKeys.length > 0) {
+      let moderationMessage = [
+      'you joined via a moderation key, meaning you are allowing someone else to help administer moderation on your behalf.', 
+      'if you would like to remove the applied moderation keys, type:'
```

Comment:
Nice! Glad you added this. There are a couple of caveats here to note or address:
- this will not undo the effects of their moderation (we really need a `/regen-moderation` command or something to recreate the mod view)
- unless you remove these keys from the cabal uri you joined with, they will become re-admin'd next time you join

Reply:
speedy feedback, thx!

> this will not undo the effects of their moderation (we really need a /regen-moderation command or something to recreate the mod view)

ah right. hmmmm

Reply:
commented the lines about which action to take out for the moment